File: 3-longest-substring-without-repeating-characters/3-longest-substring-without-repeating-characters.py
class Solution:
    def lengthOfLongestSubstring(self, s: str) -> int:
        longest = 0
        for i in range(0, len(s)):
            curString = ""
            for j in range(i, len(s)):
                if s[j] in curString:
                    break
                else:
                    curString += s[j]
                    if longest < len(curString):
                        longest = len(curString)
        return longest
                    
                
            
        

File: 3-longest-substring-without-repeating-characters/3-longest-substring-without-repeating-characters.java
class Solution {
    public int lengthOfLongestSubstring(String s) {
        int longest = 0;
        for(int i = 0; i < s.length(); i++){
            String curString = "";
            for(int j = i; j < s.length(); j++){
                if(curString.contains("" + s.charAt(j))){
                    break;
                } else {
                    curString += s.charAt(j);
                    if(longest < curString.length()){
                        longest = curString.length();
                    }
                }
            }
        }
        return longest;
    }
}